docs/contributing/styleguides/rust.md

```diff
@@ -1,3 +1,95 @@
 # Rust Style Guide
 
-Coming Soon!
\ No newline at end of file
+:zap: View the sidebar (right) for the rapid navigation.
+
+## :guardsman: Formatting, Linting and Static Analysis
+
+*TL;DR Use `rustfmt`, `cargo clippy` and copy
+[`.cargo/config.toml`](https://github.com/Arrow-air/svc-template-rust/.cargo/config.toml)*
+
+Rust does not have an official style guide. It relies on the `rustfmt` tool.
+
+Run the following before pushing code:
+- `rustfmt`: Format code to Rust's style guide
+- `cargo check`: Check a local package and all of its dependencies for errors
+- `cargo test`: Run unit and integration tests
+- `cargo clippy`: A collection of lints to catch common mistakes and improve your Rust code. [Source Code](https://github.com/rust-lang/rust-clippy)
```

Comment:
These commands can be chained. And can be built into husky.

Reply:
Add language regarding husky, also added husky to the svc-template-rust repo: https://github.com/Arrow-air/website/pull/41/commits/a91a4bab292b53abdab1f1113fd1b17d08d36248